stack_master.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "table_print"
   spec.add_dependency "dotgpg"
   spec.add_dependency "deep_merge"
+  spec.add_dependency "cfndsl"
```

Comment:
It seems that we will need to "break out" of bundler to be able to load gem X without having that in our gemspec. I think at this point it's not worth the cost. I suggest we keep doing what we are doing already (having hard dependencies).

If we want to pursue this path though, I could look at using something like [this](https://github.com/janlelis/debundle.rb/blob/modern/debundle.rb) That'll allow us to break out of bundler jail. Which might invite other undesirable issues. Thoughts?